source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -164,36 +164,73 @@ Then import the JavaScript file before the closing `</body>` tag of your HTML pa
 </body>
 ```
 
+#### Select and initialise an individual component
+
+You can select and initialise a specific component by using its `data-module` attribute. For example, initialise the first radio component on a page using `govuk-radios`:
+
+```html
+  <script>
+    var Radios = window.GOVUKFrontend.Radios
+    var $radio = document.querySelector('[data-module="govuk-radios"]')
+    if ($radio) {
+      new Radios($radio).init()
+    }
+  </script>
+```
+
```

Comment:
This was an existing section which I've moved to be nested underneath instructions on how to 'Add the JavaScript file to your HTML' because it seems more relevant there, especially as the section on importing using bundlers already contains code examples for initialising individually. Open to any other suggestions.